src/ORM/Query.php

```diff
@@ -793,6 +930,10 @@ public function __clone()
      */
     public function count()
     {
+        return $this->getCount();
+    }
+
+    public function getCount() {
```

Comment:
I accidentally committed this part as well. However, do we want to have getCount() all all? I've got the impression that some methods are named on purpose like their function names in the DB systems. So do we want to use getters here as well?